entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eu # Increase bash strictness
+
+if [[ -n "${GITHUB_WORKSPACE}" ]]; then
+  cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit
+fi
+
+export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
+
+export REVIEWDOG_VERSION=v0.11.0-nightly20210104+207b245
+
+echo "[action-flake8] Installing reviewdog..."
+wget -O - -q https://raw.githubusercontent.com/reviewdog/nightly/master/install.sh | sh -s -- -b /tmp/ "${REVIEWDOG_VERSION}"
+
+if [[ "$(which flake8)" == "" ]]; then
+  echo "[action-flake8] Installing flake8 package..."
+  python -m pip install --upgrade flake8
+  echo "[action-flake8] Flake8 version:"
+  flake8 --version
+fi
+
+# NOTE: Used for debugging
```

Comment:
It's fine, but cannot we use INPUT_FLAKE8_ARGS for debugging as well?

Reply:
That's true the only downside is that then people can not use it without using the official action. This however I think is desired behavoir.